### ISSUE TITLE
Nested list parents don't support embedded markup

### ISSUE THREAD

**Issue by 3rwolff** [puzzle/prawn-markup#41]
Nested lists don't support embedded html. Any ordered/unordered list element that has a nested list element below it, will not support the markup such as <b>, <em>, <strong>. These tags will be converted to HTML entities (i.e.  `<b>` will appear as `&lt;b&gt;`). It appears that the symptomatic elements are being converted from `strong` to `b` tags for some reason (similar for italics which also have this issue).

With the following rich text:
```
"<div class=\"trix-content\">\n  <ol>\n<li> <strong>HELLO</strong><ol>\n<li> <strong>WORLD</strong><ol>\n<li> <strong>STUFF</strong>\n</li>\n<li> <strong>STUFF2</strong>\n</li>\n</ol>\n</li>\n<li> <strong>WORLD2</strong>\n</li>\n</ol>\n</li>\n<li> <strong>HELLO2</strong>\n</li>\n</ol>\n</div>\n"
```
The output will appear as:
```
1. <b>HELLO</b>
   1. <b>WORLD</b>
        1.STUFF
        2.STUFF2
   2.WORLD2
2.HELLO2
```

**Comment by 3rwolff** [comment]
It seems that within the gem's file `lib/prawn/markup/builders/list_builder.rb:161 the defaults for `inline_format` are not set to `true` by default for lists. That seems crazy as they are for tables and blocks.

You can modify the gem's method with the following:

```
 def default_list_options
          {
            content: { inline_format: true },
            bullet: { align: :right }
          }
 end
```

For users experiencing this issue, you can use this simple workaround directly from your rails app. Simply  call `prawn.markup` with the following option:

```
 prawn.markup(text.to_s,
       {
           list: { content: { inline_format: true } }
       }
 )
```

**Comment by codez** [comment]
I could not reproduce your issue. For nested lists, this behavior was fixed in 10a03502bc43532ba0559caa64e37998397194e3
